datajoint/fetch.py

```diff
@@ -24,6 +27,16 @@ def to_dicts(recarray):
         yield dict(zip(recarray.dtype.names, rec.tolist()))
 
 
+def _flatten_attribute_list(expression, attr):
```

Comment:
I'll find this function a bit more general and useful (while I know it's a private function) if it takes in list of primary key as the first argument instead of the `expression` object. The only reason we are passing in `expression` is to grab `primary_key`.

Reply:
right.